recipes/librdkafka/all/patches/0002-Change-library-targets-and-result-variables-1-5-0.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 3c49a902..1df36a9a 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -209,9 +209,8 @@ if(WITH_ZLIB)
+ endif()
+ 
+ if(WITH_ZSTD)
+-  target_link_libraries(rdkafka PUBLIC ${ZSTD_LIBRARY})
+-  target_include_directories(rdkafka PUBLIC ${ZSTD_INCLUDE_DIR})
+-  message(STATUS "Found ZSTD: ${ZSTD_LIBRARY}")
++  target_link_libraries(rdkafka PUBLIC ${zstd::zstd})
++  message(STATUS "Found ZSTD: ${zstd_LIBRARY}")
+ endif()
+ 
+ if(WITH_SSL)
+@@ -224,7 +223,6 @@ if(WITH_SSL)
+     add_dependencies(rdkafka bundled-ssl)
+   else()
+     find_package(OpenSSL REQUIRED)
+-    target_include_directories(rdkafka PUBLIC ${OPENSSL_INCLUDE_DIR})
```

Comment:
No need to remove this line. If `${OPENSSL_INCLUDE_DIR}` is not defined, this instruction silently fails.